rollup.config.js

```diff
@@ -1,34 +1,27 @@
-const {MINIFY} = process.env;
-const minified = MINIFY === 'true';
-const outputFile = minified ? 'dist/mapbox-gl-draw.js' : 'dist/mapbox-gl-draw-unminified.js';
-
-import commonjs from '@rollup/plugin-commonjs';
-import resolve from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
 import terser from '@rollup/plugin-terser';
 
+const { MINIFY } = process.env;
+const minified = MINIFY === 'true';
+const outputFile = minified
+  ? 'dist/mapbox-gl-draw.js'
+  : 'dist/mapbox-gl-draw-unminified.js';
+
 export default {
-  input: ['index.js'],
+  input: 'index.ts',
   output: {
     name: 'MapboxDraw',
     file: outputFile,
-    format: 'umd',
+    format: 'esm',
```

Comment:
It seems like a breaking change for customers who use this library through a script tag.